### PR TITLE
*: update grpcio and prost

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -16,22 +16,17 @@ prost-codec = ["prost", "prost-derive", "lazy_static", "protobuf-build/grpcio-pr
...
 [dependencies]
 protobuf = "=2.8.0"
-prost = { version = "0.7", optional = true }
-prost-derive = { version = "0.7", optional = true }
+prost = { version = "0.9", optional = true }
+prost-derive = { version = "0.9", optional = true }
 futures = "0.3.5"
 raft-proto = { version = "0.7.0", default-features = false }
-grpcio = { version = "0.9.0", default-features = false, features = ["secure"] }
+grpcio = { version = "0.10.0", default-features = false, features = ["openssl-vendored"] }
```
Comment (sticnarf) [inline Cargo.toml:23]: Why add `openssl-vendored` in kvproto? To my understanding, the feature replacing `secure` is `boringssl`.

**Review by sticnarf — APPROVED** [review]
LGTM

**Comment by BusyJay** [comment]
I just realize that kvproto doesn't need secure features at all. The user of kvproto can enable secure features on grpcio when necessary.

**Comment by sticnarf** [comment]
Ah, right.

**Pull request merged**